src/main/java/org/havenapp/main/SettingsFragment.java

```diff
@@ -384,6 +380,9 @@ else if (!heartbeatActive && isMonitoring)
                 try {
                     String text = ((EditTextPreference) findPreference(PreferenceManager.HEARTBEAT_MONITOR_DELAY)).getText();
                     int notificationTimeMs = Integer.parseInt(text) * 60000;
+                    if (notificationTimeMs <= 0)
```

Comment:
Smallest possible input value from field is 1 minute intervals. Not strictly enforcing 5 min delay as stated